libgeotiff/configure.ac

```diff
@@ -211,16 +211,18 @@ if test "$with_libtiff" != "" ; then
 libgeotiff. Please upgrade or use an older version of libgeotiff.]),-lm)
   LIBS="$LIBS_SAVED"
 else
-  AC_CHECK_LIB(tiff,TIFFOpen,[TIFF_CONFIG=yes],
-               AC_MSG_ERROR([You will need to substantially rewrite libxtiff to
+  PKG_CHECK_MODULES(LIBTIFF, libtiff-4,
+                    [TIFF_INC="$LIBTIFF_CFLAGS" LIBS="$LIBS $LIBTIFF_LIBS" TIFF_CONFIG="yes"],
+                    [AC_CHECK_LIB(tiff,TIFFOpen,[TIFF_CONFIG=yes],
+                                  AC_MSG_ERROR([You will need to substantially rewrite libxtiff to
 build libgeotiff without libtiff]),-lm)
-  LIBS_SAVED="$LIBS"
-  AC_CHECK_LIB(tiff,TIFFMergeFieldInfo,[TIFF_CONFIG=yes],
-               AC_MSG_ERROR([Libtiff 3.6.0 Beta or later required for this version of
+                    LIBS_SAVED="$LIBS"
+                    AC_CHECK_LIB(tiff,TIFFMergeFieldInfo,[TIFF_CONFIG=yes],
```

Comment:
as you check for libtiff-4 above (so libtiff >= 4), this check testing for libtiff >= 3.6 becomes useless

Reply:
This check won't be run if pkg-config fails to find tiff >=4 however if pkg-config is not installed or pkg-config does not find tiff >=4, I kept the current AC_CHECK_LIB as a fallback and so this check will be run. So I don't think it's completely useless. However, I can remove it if you want.

Reply:
oh sorry, I didn't realize all this code was the 'else' clause of a successful PKG_CHECK_MODULES() test. Looks good then.